lab_1/start.py

```diff
@@ -12,7 +12,9 @@
     stop_words = []
 
     #  here goes your logic: calling methods from concordance.py
-
-    RESULT = None
+    concordance = main.get_concordance(['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy',

```

Comment:
that is good, but in this task you have to call all your written functions, so please, call them all and get the result